extractor/src/main/java/org/schabi/newpipe/extractor/stream/Description.java

```diff
@@ -0,0 +1,46 @@
+package org.schabi.newpipe.extractor.stream;
+
+import static org.schabi.newpipe.extractor.ServiceList.PeerTube;
+import static org.schabi.newpipe.extractor.ServiceList.YouTube;
+
+public class Description {
+    private String content;
+    private int type;
+
+    public static final int HTML = 1;
+    public static final int MARKDOWN = 2;
+    public static final int PLAIN_TEXT = 3;
+    public static final Description emptyDescription = new Description(PLAIN_TEXT, "");
+
+    public Description(int serviceID, String content) {
```

Comment:
We don't need this constructor. `Description(String content, int type)` should be enough.